data/bubaile.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "不摆了",
+  "description": "《不摆了》是一档朋友之间聊天，关于一切的播客。「Technology is the campfire around which we tell our stories」我们希望成为那堆篝火。",
+  "author": "",
```

Comment:
This isn't too big a deal, but I'd like to see author(s) attributed properly to account for their work. Once that's done we can go ahead and merge this.